WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -710,7 +710,7 @@ private extension NotificationDetailsViewController {
         }
 
         cell.onEditClick = { [weak self] _ in
-            self?.displayCommentEditorWithBlock(commentBlock)
+            wpAssertionFailure("unused")
```

Comment:
`NoteBlockActionsTableViewCell` can also be removed, but I'm not going to touch it for now since it's in Swift – maybe later.